server/utils/util_test.go

```diff
@@ -0,0 +1,53 @@
+package utils
+
+import (
+	"server/models"
+	"testing"
+)
+
+func TestHashPassword_ValidInput(t *testing.T) {
+	testPass := "123"
+	testUser := models.User{
+		Nusnetid: "e123",
+		Password: testPass,
+	}
+	err := HashPassword(&testUser)
```

Comment:
No good to print the password. Please use the username only.

Reply:
I don't quite get what you mean here, the error message below this does not print the password

Reply:
See line 19 and you will understand

Reply:
Sorry if I am missing out anything obvious but what I am seeing is that line 19 only prints out the err.Error() message from `HashPassword`, which only passes out the `err` from `bcrypt.GenerateFromPassword()` ?

Reply:
I read the code too fast. My apologies.